### DISCUSSION TITLE
Is there an example of EHCI usage with T4.x

### DISCUSSION THREAD

**Discussion by wwatson4506** [hathach/tinyusb#1429] [General]
I have always been interested in the tinyusb and EHCI usage on a T4.x. I was wondering if there is an example of its direct usage with the T4.x or Tennsy Micromod?

**Comment by hathach** [comment]
All host examples should run fine on teensy 4.0 since it is a supported board. Though I don’t personally test with it though. I mainly test host stack with lpc1857 which uses the very same usb controller as imxrt on teensy 4

**Reply by wwatson4506** [reply]
Thanks for replying. I will checkout the lpc1857 port.
